kore/app/exec/Main.hs

```diff
@@ -439,7 +449,7 @@ koreProve execOptions proveOptions = do
                     Bounded.Failed final -> return (failure final)
             else
                 either failure (const success)
-                <$> prove stepLimit mainModule specModule
+                <$> prove stepLimit mainModule specModule (Proxy :: Proxy (OnePathRule Variable))
```

Comment:
Although I don't like requiring `TypeApplication` in the interface of a function, I think it's good to make using `Proxy` less verbose:

```suggestion
                <$> prove stepLimit mainModule specModule (Proxy @(OnePathRule Variable))
```